decred/decred/dcr/vsp.py

```diff
@@ -1,22 +1,23 @@
 """
 Copyright (c) 2019, Brian Stafford
-Copyright (c) 2019, the Decred developers
+Copyright (c) 2019-2020, the Decred developers
```

Comment:
Right, we have to update that too. Hopefully only when a file is changed, though.

Reply:
I'm not sure that it actually holds any legal meaning, but might as well continue.